test/unit/spec/v2/view-builder/utils/LinksUtil_spec.js

```diff
@@ -39,4 +48,56 @@ describe('v2/utils/LinksUtil', function() {
       });
     });
   });
+
+  describe('getFactorPageCustomLink', () => {
+    it('returns a link when it is in select-authenticator-authenticate, and not a password recover follow', function() {
```

Comment:
nit: `... not a password recover **follow**` is this meant to be "flow" or "follow"?